packages/flutter/lib/src/widgets/route_notification_messages.dart

```diff
@@ -14,6 +15,21 @@ class RouteNotificationMessages {
   // ignore: unused_element
   RouteNotificationMessages._();
 
+  /// Notifies the platform for a route information change.
+  ///
+  /// See also:
+  ///
+  ///  * [SystemChannels.navigation], which handles subsequent navigation
+  ///    requests.
+  static void notifyRouteInformationChange(RouteInformation routeInformation) {
+    SystemChannels.navigation.invokeMethod<void>(
```

Comment:
The call to invokeMethod itself should be in the services library (see e.g. the static methods in `system_chrome.dart`)